darwin/importer/importer.py

```diff
@@ -957,7 +957,7 @@ def import_annotations(  # noqa: C901
 
     def import_annotation(parsed_file):
         image_id = remote_files[parsed_file.full_path]["item_id"]
-        default_slot_name = remote_files[parsed_file.full_path]["slot_names"]
+        default_slot_name = remote_files[parsed_file.full_path]["slot_names"][0]
```

Comment:
Bonus bug fix when importing annotation files that don't specify a slot name. This is one of the bugs we would have caught with E2E failures